packages/@adobe/spectrum-css-temp/components/steplist/index.css

```diff
@@ -131,137 +74,177 @@ governing permissions and limitations under the License.
-  position: absolute;
-  right: calc(var(--spectrum-steplist-step-width) - calc(var(--spectrum-steplist-marker-diameter) * 1.5));
-
-  box-sizing: content-box;
-  width: calc(calc(var(--spectrum-steplist-step-width) * 1.5) - calc(var(--spectrum-steplist-marker-diameter) * 2));
-  bottom: calc(calc(var(--spectrum-steplist-segment-height) / 2) * -1);
-
-  /* Default is dashed */
-  border-bottom-width: var(--spectrum-steplist-segment-height);
-  border-bottom-style: dashed;
+/* small size */
+.spectrum-Steplist--small {
+  .spectrum-Steplist-item {
+    grid-template-columns: calc(var(--spectrum-steplist-marker-diameter-small) * 3) auto;
+    margin-left: calc(var(--spectrum-steplist-marker-diameter-small));
+  }
+  .spectrum-Steplist-item > a {
+    grid-template-columns: calc(var(--spectrum-steplist-marker-diameter-small) * 3) auto;
+    
+    .spectrum-Steplist-marker {
+      line-height: var(--spectrum-steplist-marker-diameter-small);
+      width: var(--spectrum-steplist-marker-diameter-small);
+      height: var(--spectrum-steplist-marker-diameter-small);
+      border-radius: var(--spectrum-steplist-marker-diameter-small);
+      font-size: var(--spectrum-global-dimension-font-size-100);
+    }
+  }
 }
 
-/* First step overrides */
-.spectrum-Steplist-item:first-child {
-  .spectrum-Steplist-markerContainer {
-    left: 0;
-  }
 
-  .spectrum-Steplist-label {
-    left: 0;
+/* Medium size */
+.spectrum-Steplist--medium {
+  .spectrum-Steplist-item {
+    grid-template-columns: calc(var(--spectrum-steplist-marker-diameter-medium) * 1.5) auto;
+    margin-left: calc(var(--spectrum-steplist-marker-diameter-medium) / 2);
   }
+  
+  .spectrum-Steplist-item > a {    
+    grid-template-columns: calc(var(--spectrum-steplist-marker-diameter-medium) * 2) auto;
 
-  .spectrum-Steplist-segment {
-    /* Not visible for first step */
-    display: none;
+    .spectrum-Steplist-marker {
+      line-height: var(--spectrum-steplist-marker-diameter-medium);
+      width: var(--spectrum-steplist-marker-diameter-medium);
+      height: var(--spectrum-steplist-marker-diameter-medium);
+      border-radius: var(--spectrum-steplist-marker-diameter-medium);
+      font-size: var(--spectrum-global-dimension-font-size-75);     
+    }
   }
 }
 
-/* Last step overrides */
-.spectrum-Steplist-item:last-child {
-  .spectrum-Steplist-label {
-    left: auto;
-    right: 0;
-
-    transform: translateX(50%);
-  }
 
-  .spectrum-Steplist-markerContainer {
-    left: auto;
-    right: 0;
+/* Large size */
+.spectrum-Steplist--large {
+  line-height: var(--spectrum-steplist-marker-diameter-large);
 
-    margin-left: 0;
-    margin-right: calc(calc(var(--spectrum-steplist-marker-hitArea) / 2) * -1);
+  .spectrum-Steplist-label {
+    font-size: var(--spectrum-global-dimension-font-size-150);
   }
-
-  .spectrum-Steplist-segment {
-    right: var(--spectrum-steplist-marker-diameter);
-    left: auto;
+  .spectrum-Steplist-chevron {
+    transform: scale(1.25);
```

Comment:
we'll have to sort out how design wants to do this, but i don't think they're going to want this kind of a scale. fine for now i think